### PR TITLE
Stop gracefully on SIGTERM

### LINKED ISSUE #445: Kubernetes support: exit cleanly on SIGTERM
When running a FastTrackML server in Kubernetes and the pod gets terminated, we don't exit cleanly, which causes the SQLite WAL file to not be removed.

### PR DESCRIPTION
**Pull request by jgiannuzzi**
Fixes #445.

### CODE REVIEW

File: pkg/cmd/server.go [hunk 1]
```diff
@@ -125,6 +127,9 @@ func serverCmd(cmd *cobra.Command, args []string) error {
 	go func() {
 		sigint := make(chan os.Signal, 1)
 		signal.Notify(sigint, os.Interrupt)
+		if runtime.GOOS != "windows" {
+			signal.Notify(sigint, syscall.SIGTERM)
```
Comment (dsuhinin) [inline pkg/cmd/server.go:131]: why you can't just use:
```
signal.Notify(sigint, os.Interrupt,syscall.SIGTERM )
```
instead? os one of this events will trigger shutdown.

Comment (jgiannuzzi) [reply pkg/cmd/server.go:131]: Windows has no concept of signals and so this wouldn't be portable AFAIK

Comment (dsuhinin) [reply pkg/cmd/server.go:131]: yes yes, I know, but it won't just work in that case?

Comment (dsuhinin) [reply pkg/cmd/server.go:131]: I mean you can just use both of them in one `signal.Notify` and one notification will be triggered based on current system. this is what I meant.

Comment (jgiannuzzi) [reply pkg/cmd/server.go:131]: hmm, I thought we would get errors when using `signal.Notify` on signals that don't exist on the runtime platform, but I just tested it on Windows and it works!

**Review by dsuhinin — APPROVED** [review]

**Pull request merged**